fastlane/Fastfile

```diff
@@ -20,26 +20,27 @@ platform :ios do
     # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
     # cocoapods
     carthage(
-      platform: 'iOS',
+      platform: 'iOS,tvOS',
       cache_builds: true
     )
   end
 
   desc "Runs all the tests"
   lane :test do
-    scan(scheme: "Clappr-Example")
-    xcov(scheme: "Clappr-Example")
+    scan(scheme: "Clappr_Example", output_directory: "fastlane/test_output_ios")
```

Comment:
I believe that we work more on the iOS, can we invert the order here? tvOS and then iOS? Doing that the code coverage that is show at the end will be the one that iOS has. What do you guys think?